packages/components/cascader/src/cascader.vue

```diff
@@ -0,0 +1,658 @@
+<template>
+  <el-tooltip
+    ref="tooltipRef"
+    :visible="popperVisible"
+    :teleported="teleported"
+    :popper-class="[nsCascader.e('dropdown'), popperClass]"
+    :popper-options="popperOptions"
+    :fallback-placements="[
+      'bottom-start',
+      'bottom',
+      'top-start',
+      'top',
+      'right',
+      'left',
+    ]"
+    :stop-popper-mouse-event="false"
+    :gpu-acceleration="false"
+    placement="bottom-start"
+    :transition="`${nsCascader.namespace.value}-zoom-in-top`"
+    effect="light"
+    pure
+    persistent
+    @hide="hideSuggestionPanel"
+  >
+    <template #default>
+      <div
+        v-clickoutside:[cascaderPanelRef]="() => togglePopperVisible(false)"
+        :class="[
```

Comment:
## Suggest

How about separating that part with `computed`?

### Reference

https://github.com/element-plus/element-plus/pull/10888/files#diff-ea5ffcb1d7a99d505b8cbe4d771f599f7faccb516c33fe8874de354bedc3a4ffR83-R91

Reply:
It was originally planned to refactor the syntax first, and then put it into the document modification process.